build/root/usr/local/bin/run

```diff
@@ -192,6 +209,19 @@ case ${1:-} in
         toolbox/gpu-operator/undeploy_from_operatorhub.sh
         exit 0
 	;;
+    "sro_test-master-branch")
+        CI_IMAGE_SRO_COMMIT_CI_REPO="${2:-https://github.com/openshift-psap/special-resource-operator.git}"
+        CI_IMAGE_SRO_COMMIT_CI_REF="${3:-master}"
+
+        echo "Using Git repository ${CI_IMAGE_GPU_COMMIT_CI_REPO} with ref ${CI_IMAGE_GPU_COMMIT_CI_REF}"
+
+        prepare_cluster_for_sro
```

Comment:
(not the right location by I can't comment on `human_log.cpython-36.pyc`)

you may want to add ```
#Python
*.py[co]
```
to `~/.gitignore` or `.gitignore` :)